### DISCUSSION TITLE
video11

### DISCUSSION THREAD

**Discussion by mcharlie9** [jacekschae/learn-pedestal-course-files#6] [Feedback]
At 7:19, after adding (server/create-system), the rest of the original codes underneath it are just removed, without any explanation. This might give some students the impression that (server/create-system) on its own can cover what the removed codes did.  However, when starting the system again, what happens is that the systemMap is ready but nothing is really started yet. Therefore, it may be helpful to explain what (server/create-system) is preparing for and that the next step is to configure Component to start the system which will be done in the next video.

It is worth noting that at the end of the video, you did mention something like that. On top of that, the next video does help get things on track by stating that "our system is completely broken now", referring to the pedestal.http/http. Though not directly explaining the work-in-progress of (start-dev), it should make student comfortable with what they have been doing so far. 

Anyways, if you think this is worth refining, my suggestion is:

- to add some explanation while you are removing the above-mentioned codes OR
- to make a new concluding remark at the end of video (merging the current concluding remark with the "first 23 seconds" of video12) OR
- to add a remark in the show notes in case any students get confused.

**Comment by jacekschae** [comment]
Decided to go with point 2 - clarify at the end. I think it's very good remark and I'll try to keep this in mind, when reworking things.
